samples/python/reactors/PorousMediaBurner.py

```diff
@@ -52,8 +48,9 @@
 radiation. For more realistic and quantitative predictions, running 1D simulations with
 full radiation transport is required.
 
-More details about the governing equations, submodels and physical setup and
-corresponding experiments can be found in
+Initial code by Thorsten Zirwes and Guillaume Vignat, Stanford University & Karlsruhe
```

Comment:
Another example for inconsistent author acknowledgements. In this case, the citation should suffice.